hawc/apps/assessment/templates/assessment/assessmentvalue_detail.html

```diff
@@ -65,6 +65,15 @@ <h3>Assessment Value for {{object.assessment}}</h3>
       {% endfor %}
     </tbody>
   </table>
+  {% if pod_message %}
```

Comment:
we don't need to invite this again. Use `{% alert warning %}` and `{% endalert %}`